vignettes/sir_models.Rmd

```diff
@@ -206,10 +206,10 @@ This can be given as a parameter to the model by adding the argument `data = lis
 N_age <- length(age.limits)
 n_particles <- 5L
 dt <- 0.25
-model <- gen_age$new(data = list(dt = dt, 
+model <- gen_age$new(data = list(dt = dt,
                                  S_ini = contact$demography$population,
-                                 I_ini = rep(10,8), 
-                                 beta = 0.2, 
+                                 I_ini = c(0,10,0,0,0,0,0,0),
+                                 beta = 0.2 / 12.11,
```

Comment:
Where does this number come from? It's lots smaller than before! And different to the 0.0165 in the source (by a factor of 10)

Reply:
It's similar to the Rt calculation in sircovid (which I learnt is required in any model with a contact matrix!). Leading eigenvalue of the demography scaled contact matrix =~ 12.11. This is similar to the 'magic number' approach I think we used in early sircovid where there were two betas and no significant depletion of susceptibles, and we just rescaled the betas with a constant to get R0. Links to paper explaining it are in the vignette afterwards.

Here it's used to get plots for a transmission process which looks similar to the simple case, otherwise they have a massive R0 (=~ 24). Getting it exact probably doesn't matter too much as it's really just for the plot in the vignette, so we could just use 12 exactly to match the source. 0.2 / 12.11 = 0.01652893
(not sure where the factor of 10 is, am I missing it?)

Reply:
Nope, my fault. Not sure how I managed to do that :facepalm: